### PR TITLE
Org.gnome.crosswords

### PR DESCRIPTION
**Pull request by blandford**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [s] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.gnome.Crosswords.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "crosswords",
+    "finish-args" : [
+        "--device=dri",
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--env=PATH=/app/bin:/usr/bin",
```
Comment (hfiguiere) [inline org.gnome.Crosswords.json:13]: You shouldn't need it.
```suggestion
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build org.gnome.Crosswords

File: org.gnome.Crosswords.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "cleanup" : [
+        "/include",
+        "/lib/girepository-1.0",
+        "/lib/pkgconfig",
+        "/libexec/installed-tests",
+        "/man",
+        "/share/applications/org.gnome.Adwaita1.Demo.desktop",
+        "/share/metainfo/org.gnome.Adwaita1.Demo.metainfo.xml",
+        "/share/icons/hicolor/symbolic/apps/org.gnome.Adwaita1.Demo-symbolic.svg",
+        "/share/icons/hicolor/scalable/apps/org.gnome.Adwaita1.Demo.svg",
```
Comment (bilelmoussaoui) [inline org.gnome.Crosswords.json:24]:
```suggestion
```
Are those actually needed? You don't ship adwaita yourself as it is part of the sdk

**Comment by barthalion** [comment]
bot, build org.gnome.Crosswords

File: org.gnome.Crosswords.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "add-extensions": {
+        "org.gnome.Crosswords.PuzzleSets": {
```
Comment (barthalion) [inline org.gnome.Crosswords.json:29]: Do you plan to submit additional puzzle sets eventually?

**Comment by blandford** [comment]
Yes! I have two puzzle sets ready to go now, and I'm working on a third.
I'm also working on documentation to let 3rd-party folks produce them,
though I don't expect that immediately.

**Pull request closed without merging**